.snippets/code/tutorials/polkadot-sdk/parachains/build-custom-pallet/build-pallet/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "custom-pallet"
+version = "0.1.0"
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+edition.workspace = true
+
+[dependencies]
+codec = { features = [
+	"derive",
+], workspace = true }
+scale-info = { features = [
+	"derive",
+], workspace = true }
+frame-support.workspace = true
+frame-system.workspace = true
+
+[features]
+default = ["std"]
+std = [
+	"codec/std",
+	"frame-support/std",
+	"frame-system/std",
+	"scale-info/std",
+]
```

Comment:
Is this how the formatter formatted this file? Or was it skipped?

Reply:
This is the format applied in the `pallet-template` Cargo.toml. I just copied that formatting. https://github.com/paritytech/polkadot-sdk-solochain-template/blob/master/pallets/template/Cargo.toml

Reply:
Now it did